### PR TITLE
[fix][doc] remove `This example uses Pulsar x.x.x`

### PR DESCRIPTION
**Pull request by labuladong**
Fixes [comment in #17499 ](https://github.com/apache/pulsar/pull/17499#issuecomment-1239161837)

### Modifications

Remove redundant `This example uses Pulsar x.x.x`

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `doc-not-needed` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

cc @Anonymitaet @tisonkun @momo-jun

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]
It's more clear to be a dedicated patch now! LGTM.

File: site2/docs/tiered-storage-aliyun.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Follow the steps below to install the Aliyun OSS offloader.
 
 - Pulsar: 2.8.0 or later versions
 
-This example uses Pulsar 2.8.0.
+### Step
```
Comment (momo-jun) [inline site2/docs/tiered-storage-aliyun.md:17]:
```suggestion
### Steps
```

File: site2/docs/tiered-storage-aws.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Follow the steps below to install the AWS S3 offloader.
 
 - Pulsar: 2.4.2 or later versions
 
-This example uses Pulsar 2.5.1.
+### Step
```
Comment (momo-jun) [inline site2/docs/tiered-storage-aws.md:17]:
```suggestion
### Steps
```

File: site2/docs/tiered-storage-azure.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ Follow the steps below to install the Azure BlobStore offloader.
 
 - Pulsar: 2.6.2 or later versions
 
-This example uses Pulsar 2.6.2.
+### Step
```
Comment (momo-jun) [inline site2/docs/tiered-storage-azure.md:17]:
```suggestion
### Steps
```

File: site2/docs/tiered-storage-filesystem.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ This section describes how to install the filesystem offloader.
 
 - Pulsar: 2.4.2 or higher versions
 
-This example uses Pulsar 2.5.1.
+### Step
```
Comment (momo-jun) [inline site2/docs/tiered-storage-filesystem.md:23]:
```suggestion
### Steps
```

File: site2/docs/tiered-storage-gcs.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ Follow the steps below to install the GCS offloader.
 
 - Pulsar: 2.4.2 or later versions
 
-This example uses Pulsar 2.5.1.
+### Step
```
Comment (momo-jun) [inline site2/docs/tiered-storage-gcs.md:17]:
```suggestion
### Steps
```

**Review by RobertIndie — APPROVED** [review]

**Review by momo-jun — APPROVED** [review]
LGTM.
@labuladong can you pls also add [fix][doc] in the PR title to align with the naming conventions?

**Comment by labuladong** [comment]
All tests passed. Could you help to merge this pr? @Anonymitaet

**Pull request merged**